README.md

```diff
@@ -23,11 +23,52 @@ Your new file structure should look like:
     - Hermes
       - src
 
+## RabbitMQ Broker
+
+### Getting setup
+
+1) Installing rabbitMQ
```

Comment:
Can go more in depth when I fix the travis.yml but I would still appreciate any feedback